tests/test_saferound.py

```diff
@@ -97,6 +97,10 @@ def test_empty_returns_empty(self):
         actual_out = iteround.saferound([], 0, topline=topline)
         self.assertListEqual(actual_out, out)
 
+    def test_integers(self):
+        out = [4, 3.24, 3.23, 6.45, 5.35, 7.34]
+        self.assertListEqual(iteround.saferound(self.in_list, 2), out)
+
```

Comment:
This is not really testing that integers are in the initial data. you'll want to create your own `in_list` in the test that includes an integer.